processing/src/main/java/org/apache/carbondata/processing/loading/CarbonDataLoadConfiguration.java

```diff
@@ -248,6 +248,23 @@ public void setSchemaUpdatedTimeStamp(long schemaUpdatedTimeStamp) {
     return type;
   }
 
+  public DataType[] getMeasureDataTypeAsDataFieldOrder() {
+    // same as data fields order
+    List<Integer> measureIndexes = new ArrayList<>(dataFields.length);
+    int measureCount = 0;
+    for (int i = 0; i < dataFields.length; i++) {
```

Comment:
instead of using two for loops, i think u can use steam

Reply:
This is base code. This file has same function with for loops.